packages/vaadin-grid/test/visual/material/grid.test.js

```diff
@@ -234,9 +235,22 @@ describe('grid', () => {
 
           // Scroll all the way to end
           element.$.table.scrollLeft = element.__isRTL ? -1000 : 1000;
-          // // Focus a row
-          element.setAttribute('navigating', '');
-          element.$.items.children[0].focus();
+
+          // If supported, use tab keypress to focus the body row
+          try {
+            // Test if sendKeys is supported
```

Comment:
I don't think we need `try` / `catch` here? Visual tests can be debugged using built-in Chrome debugger.

Reply:
Done.